content/docs/trust/trust-manager/README.md

```diff
@@ -88,18 +91,27 @@ spec:
 - `inLine` - a manually specified string containing at least one certificate
 - `useDefaultCAs` - usually, a bundle of publicly trusted certificates
 
-These sources, along with the single currently supported target type (`configMap`)
-are documented in the trust-manager [API reference documentation](./api-reference.md).
+`ConfigMap` is the default target resource type, but trust-manager also supports `Secret` as targets
+(since trust-manager v0.7.0), but that feature has to be explicitly enabled on the controller.
+
+All sources and target options are documented in the trust-manager [API reference documentation](./api-reference.md).
 
 #### Targets
 
-All `Bundle` targets are written to `ConfigMap`s whose name matches that of the `Bundle`, and every
-target has a PEM-formatted bundle included.
+All `Bundle` targets are written to `ConfigMap`s (and/or `Secret`s) whose name matches that of the
+`Bundle`, and every target has a PEM-formatted bundle included.
+
+Users can also optionally choose to write JKS/PKCS#12 formatted binary trust store(s) to the target.
+JKS is supported since trust-manager v0.5.0, while v0.7.0 add support for PKCS#12.
```

Comment:
suggestion:

```suggestion
Users can also optionally choose to write JKS/PKCS#12 formatted binary trust store(s) to targets.
JKS has been supported since v0.5.0, and PKCS#12 since v0.7.0.
```